src/NuGet.Clients/PackageManagement.VisualStudio/VSSettings.cs

```diff
@@ -13,7 +13,23 @@ namespace NuGet.PackageManagement.VisualStudio
     [PartCreationPolicy(CreationPolicy.Shared)]
     public class VSSettings : Configuration.ISettings
     {
-        private Configuration.ISettings SolutionSettings { get; set; }
+        // to initialize SolutionSettings first time outside MEF constructor
+        private Configuration.ISettings _solutionSettings;
+
+        private Configuration.ISettings SolutionSettings
```

Comment:
`Lazy<Configuration.ISettings>` will make it more complicated and more code change since it's only initialized once and later on it's always reset on solution open or close.

Is there any advantage for moving to Lazy<...> here compare to current code?